build/pipeline-releasebuild.yaml

```diff
@@ -11,65 +11,28 @@ schedules:
     include:
     - master
 
-pool:
-  vmImage: 'windows-latest'
-  demands:
-  - msbuild
-  - visualstudio
-
 variables:
   BuildPlatform: 'any cpu'
   BuildConfiguration: 'release'
-  
-steps:
-# Required for the signing task, but does not need to be used
-- task: UseDotNet@2
-  displayName: 'Use .Net Core SDK 2.x'
-  inputs:
-    version: 2.x
-
-# Unit tests require .NET 3.x
-- task: UseDotNet@2
-  displayName: 'Use .Net Core SDK 3.x'
-  inputs:
-    version: 3.x
-    
-# Use the latest .NET SDK
-- task: UseDotNet@2
-  displayName: 'Use .Net Core sdk 5.x'
-  inputs:
-    version: 5.x
-
-# Run pre-build code analysis (policheck, credscan, etc)
-- template: template-prebuild-code-analysis.yaml
-
-# Bootstrap the build
-- template: template-bootstrap-build.yaml
-
-# Nuget Restore and Build 
-- template: template-restore-build-libsandsamples.yaml
-  parameters:
-    BuildPlatform: '$(BuildPlatform)'
-    BuildConfiguration: '$(BuildConfiguration)'
-    MsalClientSemVer: $(MsalClientSemVer)
-    Solution: 'LibsAndSamples.sln'
-
-# Run Post-build code analysis (e.g. Roslyn)
-- template: template-postbuild-code-analysis.yaml
-
-# Run All Tests
-- template: template-run-all-tests.yaml
-  parameters:
-    BuildConfiguration: '$(BuildConfiguration)'
-
-
-
-# Pack and sign all of the nuget packages
-- template: template-pack-and-sign-all-nugets.yaml
-
-# Publish nuget packages and symbols to VSTS package manager.
-- template: template-publish-packages-and-symbols.yaml
-
-# Publish analysis and cleanup
-- template: template-publish-analysis-and-cleanup.yaml
-
+  FastRun: true
```

Comment:
nit: Maybe add a comment here for now to specify that this refers only to the mobile automation tests.